### ISSUE TITLE
[Static Site] Remove aria-multiselectable prop that is misused

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#5809]
## Area or Component

Accordion component in[ SimpleReport data catalog page](https://www.simplereport.gov/getting-started/public-health-departments/simplereport-data-catalog/).

## Quick Summary

The accordion component is using the aria attribute aria-multiselectable which should be use only for select components that allow for multiple selections.

### How to Replicate
Go to the page and run axe scan or inspect the DOM of the accordion component. 

## Accessibility Violation

Elements must only use allowed ARIA attributes https://dequeuniversity.com/rules/axe/4.7/aria-allowed-attr?application=AxeChrome

## Action Requested

Remove the prop`aria-multiselectable`  from the accordion component.

## Screenshots
![Screenshot 2023-05-15 at 3 11 03 PM](https://github.com/CDCgov/prime-simplereport/assets/103958711/e06de794-5544-4fda-ae4b-4e94855299da)

![Screenshot 2023-05-15 at 3 11 11 PM](https://github.com/CDCgov/prime-simplereport/assets/103958711/346462fb-50a2-48e2-b6f1-1c916d117e1c)
